packages/admin/src/pages/scheme/components/SchemeApplications.test.tsx

```diff
@@ -109,6 +109,7 @@ describe('BuildApplicationForm', () => {
   it('Should render a application form view table col', () => {
     render(
       <SchemeApplications
```

Comment:
lol these should really use a function that renders the component

Reply:
you're not wrong